packages/website/src/css/custom.css

```diff
@@ -142,6 +142,10 @@ html[data-theme='dark'] .image-link::before {
   font-size: 0.8em;
 }
 
+aside nav ul code {
+  vertical-align: baseline;
```

Comment:
Will split this change out as a proposal - it's been bugging me that inline code text in the left nav menu is a little visually lower than the plain text around it.

Reply:
#11880